packages/commercetools/theme/components/UserBillingAddress.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <p class="name">
+      {{ data.name }}
+    </p>
+
+    <p>
+      {{ data.street }}
+    </p>
+
+    <p>
+      {{ data.city }}
+      {{ data.state }}
+      {{ data.zipCode }}
+    </p>
+
+    <p>{{ data.country }}</p>
+    <p v-if="data.phoneNumber">T: {{ data.phoneNumber }}</p>
+  </div>
+</template>
+
+<script>
+import { reactive, computed } from '@vue/composition-api';
+import { getSettings } from '@vue-storefront/commercetools-api';
+import { userBillingGetters } from '@vue-storefront/commercetools';
+
+export default {
+  props: {
+    address: {
+      type: Object,
+      required: true
+    }
+  },
+
+  setup(props) {
+    const data = reactive({
```

Comment:
Has it to be `reactive`? Couldn't we have smth like that?:
```js
cons data = computed(() => ({
  firstName: userBillingGetters.getFirstName(props.address),
  lastName: userBillingGetters.getLastName(props.address),
  // ...
}))
```

Reply:
It's needed to:
1.) properly react to `prop.address` changes
2.) being able to access data between getters (eg. in `name` and `street` properties)

Reply:
Fixed